code/modules/species/outsider/vox.dm

```diff
@@ -140,13 +140,33 @@
 	blood_mask =      'icons/mob/human_races/species/vox/blood_mask_armalis.dmi'
 
 	slowdown = 1.5
-	hidden_from_codex = TRUE
+	description = "Vox Armalis are a larger, more powerful subspecies of Vox, produced in lesser numbers\
+	 by the Apex for usage in heavy labor, or as shock troops. They are larger, more powerful, and \
+	 tougher than Primalis, but much slower, and have less of an iron immune system compared to their\
+	 smaller kin."
+	codex_description = "The Vox are a hostile, deeply untrustworthy species from the edges of human space. They prey \
+	on isolated stations, ships or settlements without any apparent logic or reason, and tend to refuse communications \
+	or negotiations except when their backs are to the wall or they are in dire need of resources. There seems to be a \
+	subspecies of Vox that differ from their compatriots via larger size and heavier build. Any pictures or additional \
+	data about them can be sent to the Codex offices, Venus orbit, for monetary reward."
+	hidden_from_codex = FALSE
+
+	speech_sounds = list('sound/voice/ashriek.ogg')
+	speech_chance = 10
+
 	spawn_flags = SPECIES_CAN_JOIN | SPECIES_NO_FBP_CONSTRUCTION | SPECIES_FLAG_NO_MINOR_CUT | SPECIES_IS_WHITELISTED
-	brute_mod = 0.5
-	burn_mod = 0.5
+	brute_mod = 0.6
+	burn_mod = 0.6
+	toxins_mod = 1.2
```

Comment:
Do note that this is a nerf compared to their current stats. Intentional?

Reply:
That's intentional, seeing as their skills were buffed.